industry_restaurant/data/product_product.xml

```diff
@@ -0,0 +1,225 @@
+<odoo noupdate="1">
+    <record id="product_product_5" model="product.product">
+        <field name="name">Gift Card</field>
+        <field name="detailed_type">service</field>
+        <field name="list_price">0.0</field>
+        <field name="sale_ok" eval="False"/>
+    </record>
+    <record id="product_product_33" model="product.product">
+        <field name="name">Burger Bun</field>
+        <field name="detailed_type">product</field>
+        <field name="list_price">0.0</field>
+        <field name="sale_ok" eval="False"/>
+        <field name="image_1920" type="base64" file="industry_restaurant/static/src/binary/product_template/33-image_1920"/>
+        <field name="standard_price">0.29</field>
+    </record>
+    <record id="product_product_34" model="product.product">
+        <field name="name">Caffe Americano Hot</field>
+        <field name="list_price">2.26</field>
+        <field name="categ_id" ref="product_category_7"/>
+        <field name="pos_categ_ids" eval="[(6, 0, [ref('pos_category_2')])]"/>
+        <field name="public_categ_ids" eval="[(6, 0, [ref('product_public_category_1')])]"/>
+        <field name="is_published" eval="True"/>
+        <field name="available_in_pos" eval="True"/>
+        <field name="image_1920" type="base64" file="industry_restaurant/static/src/binary/product_template/34-image_1920"/>
+        <field name="standard_price">1.76</field>
+    </record>
+    <record id="product_product_35" model="product.product">
+        <field name="name">Caffe Latte Hot</field>
+        <field name="detailed_type">product</field>
+        <field name="list_price">4.62</field>
+        <field name="categ_id" ref="product_category_7"/>
+        <field name="pos_categ_ids" eval="[(6, 0, [ref('pos_category_2')])]"/>
+        <field name="public_categ_ids" eval="[(6, 0, [ref('product_public_category_1')])]"/>
+        <field name="is_published" eval="True"/>
+        <field name="available_in_pos" eval="True"/>
+        <field name="image_1920" type="base64" file="industry_restaurant/static/src/binary/product_template/35-image_1920"/>
+        <field name="standard_price">3.88</field>
+    </record>
+    <record id="product_product_36" model="product.product">
+        <field name="name">Chicken Burger</field>
+        <field name="detailed_type">product</field>
+        <field name="list_price">1.5</field>
+        <field name="categ_id" ref="product_category_8"/>
+        <field name="pos_categ_ids" eval="[(6, 0, [ref('pos_category_3')])]"/>
+        <field name="public_categ_ids" eval="[(6, 0, [ref('product_public_category_5')])]"/>
+        <field name="is_published" eval="True"/>
+        <field name="available_in_pos" eval="True"/>
+        <field name="image_1920" type="base64" file="industry_restaurant/static/src/binary/product_template/36-image_1920"/>
+        <field name="standard_price">1.12</field>
+    </record>
+    <record id="product_product_37" model="product.product">
```

Comment:
You buy raw materials to your stock, but don't have MRP to transform them. And you don't sell them, just buy